sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala

```diff
@@ -521,20 +521,7 @@ class HiveThriftBinaryServerSuite extends HiveThriftJdbcTest {
         conf += resultSet.getString(1) -> resultSet.getString(2)
       }
 
-      assert(conf.get("spark.sql.hive.version") === Some("1.2.1"))
-    }
-  }
-
-  test("Checks Hive version via SET") {
-    withJdbcStatement() { statement =>
-      val resultSet = statement.executeQuery("SET")
-
-      val conf = mutable.Map.empty[String, String]
-      while (resultSet.next()) {
-        conf += resultSet.getString(1) -> resultSet.getString(2)
-      }
-
-      assert(conf.get("spark.sql.hive.version") === Some("1.2.1"))
```

Comment:
change it to `spark.sql.hive.metastore.version`?

Reply:
this might need to set spark.sql.hive.metastore.version explicitly

Reply:
Just make a try?

Reply:
the first commit fails this ut checking  spark.sql.hive.metastore.version. `set` cmd only shows the changed variables, if more unit tests are needed, i can add some.